### DISCUSSION TITLE
Standardizing processes

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#229] [Team Posts]
Looking through the different repos, it seems like the different translation teams have come up with different practices. It would be beneficial to talk through these differences and adopt similar practices so translations are of a similarly high quality and it is easier to integrate. So far I've found:

* Japanese:
  * requires CircleCI tests to pass for merging
  * implements [textlint](https://github.com/textlint/textlint)
* Simplified Chinese:
  * Uses a `cn` branch instead of `master`
  * requires two reviewers
  * only allows "squash merge"

Out of these unique things I've found: the founds I think are most useful are:

* CircleCI (make sure the site still works!)
* squash merge (so the commit history isn't cluttered with WIPs)

Let me know if you have any comments, or processes you think other languages should adapt, and I can integrate it into the guide for setting up a new language fork.

**Comment by QC-L** [comment]
@tesseralis 
1. We will also require CircleCI tests to pass for merging
2. Generate a preview using [netlify.com](https://www.netlify.com/). like [babel/website](https://github.com/babel/website/pull/1945)
3. Use textLint in pre-commit

**Comment by gedeagas** [comment]
@regalius

**Comment by oleksii-polovyi** [comment]
@klymenkoo
